### PR TITLE
Update CHANGELOG.md for 3.7

### PR DESCRIPTION
**Pull request by jasonblais**
Before submitting, please confirm you've
 - [X] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [X] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [X] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**
Update CHANGELOG.md for 3.7

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
 
 ### Improvements
-- Added spell checker for English, French, German, Spanish, and Dutch.
-- Removed "Display secure content only" option it's no longer necessary.
-- Reset the window position if it's restored off-screen from minimized state.
+
+#### All Platforms
+ - Added an inline spell checker for English, French, German, Spanish, and Dutch. [#225](https://github.com/mattermost/desktop/issues/225)
+ - Removed an obsolete "Display secure content only" option, following an [upgrade of the Electron app to Chrome v56](https://github.com/electron/electron/commit/2e0780308c7ef2258422efd34c968091d7cd5b65). [#469](https://github.com/mattermost/desktop/pull/469)
+ - Refactored main app window code for ... [#470](https://github.com/mattermost/desktop/pull/470)
```
Comment (jasonblais) [inline CHANGELOG.md]: I'm not sure if there's anything we can mention here that the PR improved (e.g. performance, rendering, .... ?)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,53 +1,64 @@
...
+ - Reset app window position when restoring it off-screen from a minimized state. [#471](https://github.com/mattermost/desktop/issues/471)
+ - Improved page loading and app view rendering when waking computer from sleep.
 
 #### Windows
-- Removed Japanese fonts support it's no longer necessary.
-- Added support for sound notifications on Windows 7 and 8.
-- The application now respects 125% of display scaling.
+ - [Windows 7/8] Added support for sound when a desktop notification is received. [#467](https://github.com/mattermost/desktop/issues/467)
+ - Removed obsolete support for Japanese fonts.
+ - Removed blurriness on the app window at 125% display resolution.
```
Comment (yuya-oc) [inline CHANGELOG.md]: https://github.com/mattermost/desktop/pull/489

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,53 +1,64 @@
...
+ - Added an inline spell checker for English, French, German, Spanish, and Dutch. [#225](https://github.com/mattermost/desktop/issues/225)
+ - Removed an obsolete "Display secure content only" option, following an [upgrade of the Electron app to Chrome v56](https://github.com/electron/electron/commit/2e0780308c7ef2258422efd34c968091d7cd5b65). [#469](https://github.com/mattermost/desktop/pull/469)
+ - Refactored main app window code for ... [#470](https://github.com/mattermost/desktop/pull/470)
+ - Reset app window position when restoring it off-screen from a minimized state. [#471](https://github.com/mattermost/desktop/issues/471)
+ - Improved page loading and app view rendering when waking computer from sleep.
 
 #### Windows
-- Removed Japanese fonts support it's no longer necessary.
-- Added support for sound notifications on Windows 7 and 8.
-- The application now respects 125% of display scaling.
```
Comment (yuya-oc) [inline CHANGELOG.md]: This sentence is also necessary. The size was also changed not only blurriness.
https://github.com/mattermost/desktop/pull/459#issuecomment-283984666

File: CHANGELOG.md (hunk 1 above)
Comment (yuya-oc) [inline CHANGELOG.md]: At this time, the refactoring affects its code structure only.

File: CHANGELOG.md (hunk 3 above)
Comment (jasonblais) [inline CHANGELOG.md]: Ah, that's right. I'll change it back to that since it would cover the blurriness too

**Comment by jasonblais** [comment]
Thanks! I've updated the PR

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,53 +1,63 @@
...
 ### Improvements
-- Added spell checker for English, French, German, Spanish, and Dutch.
-- Removed "Display secure content only" option it's no longer necessary.
-- Reset the window position if it's restored off-screen from minimized state.
+
+#### All Platforms
+ - Added an inline spell checker for English, French, German, Spanish, and Dutch. [#225](https://github.com/mattermost/desktop/issues/225)
+ - Removed an obsolete "Display secure content only" option, following an [upgrade of the Electron app to Chrome v56](https://github.com/electron/electron/commit/2e0780308c7ef2258422efd34c968091d7cd5b65). [#469](https://github.com/mattermost/desktop/pull/469)
+ - Reset app window position when restoring it off-screen from a minimized state. [#471](https://github.com/mattermost/desktop/issues/471)
+ - Improved page loading and app view rendering when waking computer from sleep.
```
Comment (yuya-oc) [inline CHANGELOG.md]: Sorry, I had forgot to write about this. https://github.com/mattermost/desktop/pull/495 is still pending though, what part is the factor for this log?

Comment (jasonblais) [inline CHANGELOG.md]: I wrote that entry based on https://github.com/mattermost/desktop/pull/515

Comment (yuya-oc) [inline CHANGELOG.md]: Yeah, appropriate.

**Pull request merged**